src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
For better flexibility make storage accept parameters Fruit and Integer

Reply:
I think that it's redundant action, as in our Fruit class will be only one field 'name' and it makes the logic of parsing more complex and brings more complexity into the code. We have a good principle KISS - Keep It Simple Stupid

Reply:
Sure, but we learn to write projects using best practices
Here storage represents the database for the application, but we should always do all the logic related to working with data on DAO layer